torch/_inductor/utils.py

```diff
@@ -100,7 +100,11 @@ def do_bench_using_profiling(fn: Callable[[], Any], warmup=25, rep=100) -> float
     log.debug(p.key_averages().table(sort_by="self_cuda_time_total", row_limit=-1))
 
     filtered_events = EventList(
-        [event for event in p.events() if event.device_type == DeviceType.CUDA]
+        [
+            event
+            for event in p.events()
+            if event.device_type == DeviceType.CUDA and event.name != "Context Sync"
```

Comment:
I think there are other " Sync" events too, but I don't recall their exact names
(I drafted a similar fix for AIT where I just checked for that substring)

Reply:
In the AIT case, we don't exclude "DeviceType.CPU" so there are some CPU events like CUDASynchronize, etc. Those events don't have CUDA time. For "DeviceType.CUDA" events it seems to me that we only need to exclude "Context Sync".

Reply:
These are irrelevant events that we ended up excluding in other PT-based benchmarking:

```
    ignore_sync_events = [
        "Event Sync",
        "Stream Wait Event",
        "Stream Sync",
        "Context Sync",
        "Unknown Sync",
    ]
```

Might make sense to proactively exclude them, too.